zebra-rpc/src/server/error.rs

```diff
@@ -0,0 +1,118 @@
+//! RPC error codes & their handling.
+
+/// Bitcoin RPC error codes
+///
+/// Drawn from <https://github.com/zcash/zcash/blob/master/src/rpc/protocol.h#L32-L80>.
```

Comment:
```suggestion
/// Drawn from <https://github.com/zcash/zcash/blob/99ad6fdc3a549ab510422820eea5e5ce9f60a5fd/src/rpc/protocol.h#L32-L80>.
```

It is maybe safer to link to a commit.